io/pci/distro_tools.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Author: Bimurti Bidhibrata Pattjoshi <bbidhibr@in.ibm.com>
+#
+
+"""
+Tests the different tool
+"""
+
+from avocado import main
+from avocado import Test
+from avocado.utils import process
+
+
+class DisrtoTool(Test):
+    '''
+    to test different type of tool
+    '''
+
+    def setUp(self):
+        '''
+        to check host platform
+        '''
+        self.option = self.params.get("option", default='')
+        cmd = "cat /proc/cpuinfo | grep \'pSeries\'"
+        if process.system(cmd, shell=True, verbose=True,
+                          ignore_status=True):
+            self.cancel("Test not supported on the PowerNV Host platform")
+
+    def test(self):
+        '''
+        test the lsslot tool
+        '''
+        cmd = "lsslot"
+        if self.option == "":
+            cmd = "%s -d" % cmd
```

Comment:
why can't this be `-d` in yaml ?

Reply:
all option are comming after -c, and -d is comming lsslot -d so i did like this.